packages/tasit-action/src/contract/Contract.test.js

```diff
@@ -146,12 +153,18 @@ describe("TasitAction.Contract", () => {
 
         action = sampleContract.revertWrite("some string");
 
+        // Some error (orphan block, failed tx) events are being triggered only from the confirmationListener
+        // See more: https://github.com/tasitlabs/TasitSDK/issues/253
+        action.on("confirmation", () => {});
+
         action.on("error", actionErrorListener);
 
         await action.waitForNonceToUpdate();
 
-        expect(contractErrorListener.callCount).to.equal(1);
-        expect(actionErrorListener.callCount).to.equal(1);
+        await mineBlocks(provider, 1);
+
+        expect(contractErrorListener.called).to.be.true;
+        expect(actionErrorListener.called).to.be.true;
```

Comment:
Any reason you're moving away from `callCount`?

Reply:
Because the number of calls is being non-deterministic, I can move to `.to.be.at.least(1)` and make a comment about that, should I?

Reply:
Yeah, I think I prefer that if that works for you

Reply:
And the comment would definitely be helpful

Reply:
I think that it was addresses by the https://github.com/tasitlabs/TasitSDK/pull/358

Reply:
Yep. Thanks. Resolving this.